docs/content-crag/integrations/dbt.mdx

```diff
@@ -1,6 +1,11 @@
+---
+title: Dagster with DBT
+description: Dagster can orchestrate DBT models.
+---
+
 # Using dbt with Dagster
 
-This guide explains how you can run a [dbt](https://docs.getdbt.com/docs/introduction) project in your Dagster pipelines.
+This guide explains how you can run a [dbt](https://docs.getdbt.com/docs/introduction) project as part of a Dagster graph.
```

Comment:
should we really be communicating as graph instead of pipeline? I feel like the pipeline equivalent is more accurately job, no?

Reply:
This is the main drawback of our new ontology - "pipeline" maps to both "graph" and "job".  Is there a particular reason that you think job is more accurate in this context?